client/js/app.js

```diff
@@ -59,17 +59,31 @@ const init = () => {
     makeSticky(stickyEls, store$);
   }
 
-  if (seriesNav) {
-    shrinkStoriesNav(seriesNav, dispatch);
-  }
-
-  if (seriesSlider) {
-    contentSlider(seriesSlider, {
-      transitionSpeed: 0.7,
-      startPosition: 0,
-      cssPrefix: 'numbered-list__'
+  const asyncSeriesNavAdded$ = store$.map((state) => {
+    return state.asyncComponents.some((item) => {
```

Comment:
What I think(/I want) to be happening here is `asyncSeriesNavAdded$` to be an Observable that will complete as soon as there's one 'sticky-nav' component in the `state.asyncComponents` array (and a listener subscribed to it will set up the sticky nav logic).

I wasn't sure about having the logic in here, though – I'll see about moving it to the `asyncContent` component.